src/main/java/org/folio/rest/converter/ResourcesConverter.java

```diff
@@ -69,10 +75,53 @@ public ResourcesConverter(CommonAttributesConverter commonConverter) {
             .withManagedCoverages(commonConverter.convertCoverages(resource.getManagedCoverageList()))
             .withCustomCoverages(commonConverter.convertCoverages(resource.getCustomCoverageList()))
             .withProxy(commonConverter.convertProxy(resource.getProxy())))
-          .withRelationships(EMPTY_RELATIONSHIPS)
+          .withRelationships(createEmptyRelationship())
         )
         .withIncluded(null)
-        .withJsonapi(RestConstants.JSONAPI))
-      .collect(Collectors.toList());
+        .withJsonapi(RestConstants.JSONAPI);
+      resultResource.setIncluded(new ArrayList<>());
+      if (includeTitle) {
+        resultResource.getIncluded().add(titleConverter.convertFromRMAPITitle(title, null).getData());
```

Comment:
This comment is not related to this PR but it will be good to have consistency in naming converter methods -> convertFromRMAPITitle, convertToProvider, convert etc.